docs/modules/ROOT/pages/documentation/payara-server/server-configuration/http/protocols.adoc

```diff
@@ -13,3 +13,604 @@ commands. The protocol configurations can be found in the admin console under
+
+[[configuration-http-compression-min-size]]
+=== Compression Minimum Size
+
+The minimum size of a file when compression is applied.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.compression-min-size-bytes=value`
+
+[[configuration-http-no-compression-user-agents]]
+=== No-Compression User Agents
+
+Comma separated list of regular expressions matching user agents for which compression should not be used.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.no-compression-user-agents=value`
+
+[[configuration-http-restricted-user-agents]]
+=== Restricted User Agents
+
+List of restricted user agents for which HTTP compression is applied.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.restricted-user-agents=value`
+
+[[configuration-http-default-response-type]]
+=== Default Response Type
+
+The default response type if none are matched. Specified as a semi-colon delimited string consisting of
+content type, encoding, language, charset.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.default-response-type=value`
+
+[[configuration-http-adapter]]
+=== Adapter
+
+The class name of the static resources adapter.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.adapter=value`
+
+[[configuration-http-comet-support]]
+=== Comet Support
+
+Whether comet support is enabled or not.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.comet-support-enabled=[true/false]`
+
+[[configuration-http-dns-lookup]]
+=== DNS Lookup
+
+Enables Domain Name System lookups.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.dns-lookup-enabled=[true/false]`
+
+[[configuration-http-trace-operation]]
+=== Trace Enabled
+
+Enables the TRACE operation. Setting this to false helps prevent cross-site scripting attacks.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.trace-enabled=[true/false]`
+
+[[configuration-http-auth-pass-through]]
+=== Auth Pass Through
+
+Enable this if the network listener receives traffic from an SSL terminating proxy server.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.auth-pass-through-enabled=[true/false]`
+
+[[configuration-http-chunking-enabled]]
+=== Chunking Enabled
+
+Enables HTTP response chunking.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.chunking-enabled=[true/false]`
+
+[[configuration-http-xpowered-by]]
+=== XPoweredBy
+
+Whether to include the X-Powered-By header in responses.
```

Comment:
This description is wrong.  It should state: "Used to overwrite the default `X-Powered-By` header string"